src/Application/Common/Results/CommonErrors.cs

```diff
@@ -225,6 +225,12 @@ public static Error ItemNotOwned(string itemId) => new(ErrorType.NotFound, Error
         Detail = $"Item with id '{itemId}' is not owned by the user",
     };
 
+    public static Error ItemIsNotSellable(string itemId) => new(ErrorType.Validation, ErrorCode.ItemNotSellable)
```

Comment:
Sorry just a nitpick that I've missed but could you make the method name match the enum name 🙏